tests/WebexMeeting.e2e.js

```diff
@@ -22,84 +22,67 @@ describe('Meeting Widget', () => {
-  });
-
   it('has the correct page title', () => {
     expect(SamplesPage.widgetTitle.getText()).toBe('Webex Meeting Widget');
   });
 
   it('loads', () => {
-    MeetingPage.loadWidget(meetingDestination);
     expect(MeetingPage.meetingInfo).toBeDisplayed();
-    MeetingPage.unloadWidget();
   });
 
   it('mutes audio before joining meeting', () => {
-    MeetingPage.loadWidget(meetingDestination);
     MeetingPage.muteAudioBtn.click();
     expect(MeetingPage.muteAudioBtn).not.toBeVisible();
     expect(MeetingPage.unmuteAudioBtn).toBeVisible();
-    MeetingPage.unloadWidget();
   });
 
   it('mutes video before joining meeting', () => {
-    MeetingPage.loadWidget(meetingDestination);
     MeetingPage.muteVideoBtn.click();
     expect(MeetingPage.muteVideoBtn).not.toBeVisible();
     expect(MeetingPage.unmuteVideoBtn).toBeVisible();
-    MeetingPage.unloadWidget();
   });
 
   it('displays "Waiting for others" after joining meeting', () => {
-    MeetingPage.loadWidget(meetingDestination);
     expect(MeetingPage.waitingForOthers).not.toExist();
     MeetingPage.joinMeetingBtn.click();
     MeetingPage.waitingForOthers.waitForDisplayed({timeout: 10000});
     expect(MeetingPage.waitingForOthers).toBeVisible();
-    MeetingPage.unloadWidget();
   });
 
-  it('mutes audio after joining meeting', () => {
-    MeetingPage.loadWidget(meetingDestination);
-    MeetingPage.joinMeetingBtn.click();
-    MeetingPage.waitingForOthers.waitForDisplayed({timeout: 10000});
-    MeetingPage.muteAudioBtn.click();
-    expect(MeetingPage.muteAudioBtn).not.toBeVisible();
-    expect(MeetingPage.unmuteAudioBtn).toBeVisible();
-    MeetingPage.unloadWidget();
+  it('unmutes audio after joining meeting', () => {
+    MeetingPage.unmuteAudioBtn.click();
+    MeetingPage.muteAudioBtn.waitForDisplayed({timeout: 10000});
+    expect(MeetingPage.unmuteAudioBtn).not.toBeVisible();
+    expect(MeetingPage.muteAudioBtn).toBeVisible();
   });
 
-  it('mutes video after joining meeting', () => {
-    MeetingPage.loadWidget(meetingDestination);
-    MeetingPage.joinMeetingBtn.click();
-    MeetingPage.waitingForOthers.waitForDisplayed({timeout: 10000});
-    MeetingPage.muteVideoBtn.click();
-    expect(MeetingPage.unmuteVideoBtn).toBeVisible();
-    MeetingPage.unloadWidget();
+  it('unmutes video after joining meeting', () => {
+    MeetingPage.unmuteVideoBtn.click();
+    MeetingPage.muteVideoBtn.waitForDisplayed({timeout: 10000});
+    expect(MeetingPage.unmuteVideoBtn).not.toBeVisible();
+    expect(MeetingPage.muteVideoBtn).toBeVisible();
   });
 
   it('leave the meeting', () => {
-    MeetingPage.loadWidget(meetingDestination);
-    MeetingPage.joinMeetingBtn.click();
-    MeetingPage.waitingForOthers.waitForDisplayed({timeout: 10000});
     MeetingPage.leaveMeetingBtn.click();
     MeetingPage.meetingWidget.waitForDisplayed({timeout: 10000});
     expect(MeetingPage.meetingWidget).toHaveTextContaining("You've successfully left the meeting");
```

Comment:
@cipak left a comment on this on a previous PR, so let's address it here. Please single quotes